bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/Dispatcher.java

```diff
@@ -60,7 +61,7 @@ public Dispatcher()
     @Override
     public void onGuildMessageReceived(final GuildMessageReceivedEvent event)
     {
-        if(event.getAuthor().isBot() || event.getAuthor().isFake())
+        if(event.getAuthor().isBot() || event.getMessage().isWebhookMessage())
```

Comment:
Webhook authors are always bots so this second check is redundant

Reply:
cheers